src/definitions/collections/grid.less

```diff
@@ -1112,12 +1112,12 @@
 .ui.relaxed[class*="vertically divided"].grid > .row:before {
   margin-left: (@relaxedGutterWidth / 2);
   margin-right: (@relaxedGutterWidth / 2);
-  width: ~"calc(100% - "@relaxedGutterWidth~")";
+  width: e(%("calc(100%% - %d)", @relaxedGutterWidth));
```

Comment:
`100%%` ? This might not be correct.

Reply:
It is:
> If you need to print the percentage symbol, escape it by another percentage %%.

% format
http://lesscss.org/functions/#string-functions

Reply:
Oh yeah I see, sorry !